### ISSUE TITLE
Save shaders to ghosts, ships and sparrows in loadouts.

### ISSUE THREAD

**Issue by ryan-rushton** [DestinyItemManager/DIM#7759]
### Proposed change

A user on reddit noted that they would like to save shaders on ghosts, ships and sparrows to fully sync their fashion.

### How does this fit into your workflow?

Fashion is king.

**Comment by bhollis** [comment]
Yup, it'd be cool. It'll take some doing as right now all the mods code is very specific to armor.
